test/e2e_flags.go

```diff
@@ -78,6 +81,14 @@ func initializeEventingFlags() *EventingEnvironmentFlags {
 	flag.Var(&f.Provisioners, "clusterChannelProvisioners", "The names of the Channel's clusterChannelProvisioners, which are separated by comma.")
 	flag.BoolVar(&f.RunFromMain, "runFromMain", false, "If runFromMain is set to false, the TestMain will be skipped when we run tests.")
 
+	imagePullPolicy := string(v1.PullIfNotPresent)
+
+	if os.Getenv("IMAGE_PULL_POLICY") != imagePullPolicy {
+		imagePullPolicy = string(v1.PullAlways)
+	}
+
+	flag.StringVar(&f.ImagePullPolicy, "imagePullPolicy", imagePullPolicy, "Provide the image pull policy for the test images. Valid value is " + string(v1.PullIfNotPresent) + ", any other value will default to " + string(v1.PullAlways) + ".")
```

Comment:
To make sure I am reading this correct. Is this using the EnvVar to create a default for the flag? Then the flag is used without any validation?

Reply:
@Harwayne  I am checking the EnvVar value on [line 86](https://github.com/knative/eventing/pull/847/files#diff-b9894a959e303b07a9bf387643852461R86), the only valid value is "IfNotPresent", in any other case the imagePullPolicy is set to "Always". 
Is there any other validation I should add for the flag? Thanks.

Reply:
It seems like what you want is a way to set the image pull policy to either Always or PullIfNotPresent. Do you want to do this via a flag or an env var? As written, this code allows setting via both flag and env var. Is that intended?

There is no validation on the flag, so if I add `-imagePullPolicy foobar` (which overrides the env var), then the image pull policy that is sent to K8s will be `foobar`.

Reply:
I want to set it via env var, I don't really need flag at all. So should I get rid of the flag completely?

Reply:
My preference is also the env var, so I think removing the flag is the right thing to do.